gulpfile.js

```diff
@@ -41,3 +42,11 @@ gulp.task('prpl-server', gulp.series(
   'prpl-server:clean',
   'prpl-server:build'
 ));
+
+/**
+ * Gulp task to run `tsc --watch` and `polymer serve` in parallel.
+ */
+gulp.task('serve', () => {
+  spawn('tsc', ['--watch'], { stdio: 'inherit' });
+  spawn('polymer', ['serve'], { stdio: 'inherit' });
```

Comment:
I think this might not work on Windows. According to https://nodejs.org/api/child_process.html#child_process_spawning_bat_and_cmd_files_on_windows, 
we need to add `shell: true` option.

Reply:
I found we did something similar in polymer-analysis: https://github.com/Polymer/polymer-analyzer/blob/63fcf424eafe4420a2ddc4a97a29848405dd2dc1/gulpfile.js#L117

Reply:
Tried to run it on Windows and got this error:
<img width="949" alt="screen shot 2018-10-23 at 4 53 36 pm" src="https://user-images.githubusercontent.com/116360/47397471-3da65a80-d6e4-11e8-9d19-eeed840110da.png">

Adding `shell: true` option seems to fix it.